cityofaustin/atd-vz-data
# Clean up ETL directory and Docker image CI

## Associated issues

- https://github.com/cityofaustin/atd-data-tech/issues/18438

## Testing

1. Before you pull, you probably want to make a copy of your env file at `atd-etl/data_model/.env`
2. After you pull, delete the old `env` filed fro `./atd-etl/cris_import` and save your `.env` file there
3. Start your local stack
4. Run the cris import from `./atd-etl/cris_import`:

```shell
 docker run -it  --rm --env-file .env --network host -v $PWD:/app data_model_cris_import ./cris_import.py --csv  --pdf --s3-download --verbose
```

4. Look at the github action changes and see if those look ok 🔍
